lib/frame.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Frame
```

Comment:
This class doesn't do very much, and I'd be tempted to merge it with the FrameScore class. This is a scorecard, so I feel the only model we really need for a frame is a model around scoring.